mantidproject/fitbenchmarking
# Redesigned Unit Tests

#### Description of Work

Fixes #91 
Unit tests come now in smaller suites located within the directory with scripts that they are testing.
They are also simpler and easier to read.

#### Testing Instructions

1. Within each directory, check if there is a test folder. If there is such a folder, run the scripts within it in the `mantidpython` terminal.
2. Make sure all the tests pass.
